eventsources/sources/github/start.go

```diff
@@ -253,6 +256,15 @@ func (router *Router) PostInactivate() error {
 
 	githubEventSource := router.githubEventSource
 
+	if githubEventSource.APIToken == nil {
+		logger := router.route.Logger.With(map[string]interface{}{
```

Comment:
We changed to use `zap` as log tool, check the following code for `With` function.

Reply:
done